meeteval/io/seglst.py

```diff
@@ -43,13 +45,66 @@ class SegLST(BaseABC):
     # Caches
     _unique = None
 
+    @classmethod
+    def load(cls, file: [Path, str, io.TextIOBase, tuple, list], parse_float=decimal.Decimal) -> 'Self':
+        from meeteval.io.base import _open
+        files = file if isinstance(file, (tuple, list)) else [file]
+
+        parsed = []
+        for f in files:
+            with _open(f, 'r') as fd:
+                parsed.append(cls.parse(fd.read(), parse_float=parse_float))
+
+        return cls.merge(*parsed)
+
+    @classmethod
+    def parse(cls, s: str, parse_float=decimal.Decimal) -> 'Self':
+        """
+        Parses a SegLST from a string.
+
+        >>> SegLST.parse('[{"words": "a b c", "segment_index": 0, "speaker": 0}]')
+        SegLST(segments=[{'words': 'a b c', 'segment_index': 0, 'speaker': 0}])
+        """
+        import simplejson
+
+        def fix_floats(s):
+            """Convert common float keys to decimal"""
+            for k in ('start_time', 'end_time', 'confidence'):
+                if k in s:
+                    s[k] = decimal.Decimal(s[k])
```

Comment:
Why not
```
s[k] = parse_float(s[k])
```
?

Btw. for loading, we don't need simplejson. It is only necessary in dump to avoid a hack of the internals of the json module.

Reply:
Done